README.md

```diff
@@ -70,6 +70,10 @@ Run and test the Android sample mobile application by following the steps in [Ru
 
 Follow the steps in [Sign in users and call an API in a sample Android mobile app by using native authentication](https://learn.microsoft.com/entra/external-id/customers/sample-native-authentication-android-sample-app-call-web-api) to sign in users and call a protected API in the Android sample mobile app.
 
+### Step 10: Add email one-time passcode MFA (Private Preview)
+
+Add email one-time passcode MFA to the sign in flow by following the steps in [Add email one-time passcode MFA to your Android app](https://github.com/microsoft/entra-previews/blob/PP5/docs/Native-Auth/Developer-guides/0-Android-Kotlin/3-Add-email-otp-mfa-sign-in.md).
```

Comment:
It's important to say that we usually link documentation that talk about sample app, while email OTP MFA documentation is a tutorial about how the user can integrate email OTP MFA in their app

Reply:
Good point. Done.